meta/main.yml

```
@@ -1,10 +1,12 @@
 ---
 galaxy_info:
-  author: First Last
-  description: Skeleton Ansible role
+  author: Alexander King
+  description: Download and install the Metasploit Framework
   company: CISA Cyber Assessments
   galaxy_tags:
-    - skeleton
+    - metasploit
+    - pentest
+    - security
```

Just a thought (I saw a role to install Nexpose that uses the tag).

```suggestion
    - rapid7
    - security
```